CHANGELOG.md

```diff
@@ -7,211 +7,49 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- [#11056](https://github.com/MetaMask/metamask-extension/pull/11056): Sync `master` with `develop`
-- Fixing changelog, bumping package.json -> 9.5.1
-- Merge remote-tracking branch 'origin/develop' into master-sync
-- [#11051](https://github.com/MetaMask/metamask-extension/pull/11051): Fix Sentry sourcemaps
-- [#11049](https://github.com/MetaMask/metamask-extension/pull/11049): Use isValidAddress helper instead of ethereumJsUtil.isValidAddress, when appropriate
-- [#11034](https://github.com/MetaMask/metamask-extension/pull/11034): Bump hosted-git-info from 2.5.0 to 2.8.9
-- [#11016](https://github.com/MetaMask/metamask-extension/pull/11016): Group signature transactions
-- [#11009](https://github.com/MetaMask/metamask-extension/pull/11009): Replace uses of `hasOwnProperty.call` with simpler alternatives
-- [#11031](https://github.com/MetaMask/metamask-extension/pull/11031): Fix error behavior of addEthereumChain
-- [#11030](https://github.com/MetaMask/metamask-extension/pull/11030): Update switchEthereumChain unrecognized chain error
-- [#11029](https://github.com/MetaMask/metamask-extension/pull/11029): Migrate version from `_base` manifest to `package.json`
-- [#10905](https://github.com/MetaMask/metamask-extension/pull/10905): rough in switchEthereumChain
-- [#11025](https://github.com/MetaMask/metamask-extension/pull/11025): Fix redirection to the build quotes page from the swaps failed page
-- [#11011](https://github.com/MetaMask/metamask-extension/pull/11011): Update jsconfig excludes array
+- [#10963](https://github.com/MetaMask/metamask-extension/pull/10963): Ensure method of adding contact when contacts exist
+- [#10955](https://github.com/MetaMask/metamask-extension/pull/10955): Anchor the Need Help text to the bottom of the expanded page
+- [#10774](https://github.com/MetaMask/metamask-extension/pull/10774): Swaps: Use network specific swaps contract address when checking swap contract token approval
+
+### Fixed
+- [#11025](https://github.com/MetaMask/metamask-extension/pull/11025): Fixed redirection to the build quotes page from the swaps page when failure has occured
+- [#11015](https://github.com/MetaMask/metamask-extension/pull/11015): Prevent an undefined gas price from breaking the transaction list
 - [#11013](https://github.com/MetaMask/metamask-extension/pull/11013): Prevent signature request component from canceling hardware wallet signing
-- [#11014](https://github.com/MetaMask/metamask-extension/pull/11014): Removing unused prop from TransactionActivityLog
-- [#11012](https://github.com/MetaMask/metamask-extension/pull/11012): Use strict assertion mode everywhere
-- [#11003](https://github.com/MetaMask/metamask-extension/pull/11003): Remove `SET_CURRENT_FIAT` action
-- [#11007](https://github.com/MetaMask/metamask-extension/pull/11007): adding condition to check for non-mainnet network and test env
```

Comment:
I think https://github.com/MetaMask/metamask-extension/pull/11007 should be in this changelog

Reply:
It is right here: https://github.com/MetaMask/metamask-extension/pull/11133/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR21